tracks.yaml

```diff
@@ -62,15 +62,15 @@ tracks:
       -i :{release_inc} --os-name debian --os-not-required
     - git-bloom-generate -y rosrpm --prefix release/:{ros_distro} :{ros_distro} -i
       :{release_inc}
-    devel_branch: master
+    devel_branch: releases/0.6.x
     last_release: 004932817e111597db65e6b332f92ae9bbe9fe30
     last_version: 0.5.1
     name: cyclonedds
     patches: null
     release_inc: '2'
     release_repo_url: https://github.com/ros2-gbp/cyclonedds-release.git
-    release_tag: 004932817e111597db65e6b332f92ae9bbe9fe30
+    release_tag: :{version}
     ros_distro: foxy
     vcs_type: git
     vcs_uri: https://github.com/eclipse-cyclonedds/cyclonedds.git
-    version: 0.5.1
+    version: 0.6.0
```

Comment:
@eboasson this could be changed to `:{auto}` if you want the version listed in the devel_branch's package.xml to be used. Otherwise it will need to be manually updated before each release.